src/main/java/Ryan.java

```diff
@@ -0,0 +1,58 @@
+import java.util.Scanner;
+import java.util.ArrayList;
+
+public class Ryan {
+    public static void main(String[] args) {
+        boolean exit = false;
+        Scanner scanner = new Scanner(System.in);
```

Comment:
Would benefit from an additional line break above it for readability after variable declarations.